src/OrbitCore/SymbolHelper.h

```diff
@@ -45,6 +49,10 @@ class SymbolHelper {
       const fs::path& debug_directory, std::string_view build_id);
 
  private:
+  [[nodiscard]] static ErrorMessageOr<void> VerifySymbolsFile(
+      const fs::path& symbols_path,
+      llvm::object::OwningBinary<llvm::object::ObjectFile>&& object_file,
+      std::string_view build_id);
```

Comment:
I think this doesn't need to be a member function. I can probably declared as a free static function in the cpp file as it is only intended to be called from the other `VerifySymbolsFile`, isn't it?

Reply:
Also not needed in the new version.